### DISCUSSION TITLE
Hide or show the navbar-brand element depending on breakpoint?

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#35306] [Q&A]
Hi All, 

i'm trying to get the bootstrap navbar 'brand' element to hide or show, while attempting to hide or show another div at the same time. 

i have created a video to illustrate, so as to avoid confusing the explanation:

https://user-images.githubusercontent.com/40913310/139541803-ac5ae77d-0f66-4382-9f68-7acf66e0f806.mp4

Feels like there is a breakpoint in there somewhere that I am missing?

**Comment by adityadees** [comment]
media query will solve this problem. [w3schools]( https://www.w3schools.com/css/css_rwd_mediaqueries.asp)
